ceno_zkvm/Cargo.toml

```diff
@@ -44,6 +44,7 @@ tempfile = "3.14"
 thread_local = "1.1"
 
 [dev-dependencies]
+bincode = "1"
```

Comment:
We already use serde and rkyv, I don't think we need to introduce yet another serialisation framework?  (Perhaps just use serde?)

Reply:
Having said all that, if you want to keep bincode, you can keep it.  The only real downside I see is a slightly longer build time, but it's probably not that bad.

Reply:
If we wish to compare the size of our proof with sp1 directly, bincode would provide a bit more confident to the comparision since sp1 use bincode to serialize their proof.

And yes, we can only apply this crate as a dev-dependency